moped-database/migrations/1663082078687_adds_pd_subphases_and_updates_phase_description/down.sql

```diff
@@ -0,0 +1,11 @@
+-- reverts add subphase "Preliminary Engineering Report" to the "Preliminary Engineering" phase
+DELETE FROM moped_subphases  WHERE subphase_name ='Preliminary Engineering Report' and related_phase_id = 3;
```

Comment:
`subphase_name`'s are not guaranteed unique, so it's important that we delete on name + related phase ID.